server/playground/index.js

```diff
@@ -0,0 +1,28 @@
+const buildSchema = require("data-sync-gql-core");
+const { makeExecutableSchema } = require("graphql-tools");
+
+const { ApolloServer } = require("apollo-server-express");
+const models = require("../models");
+
+function connectDataSources(dataSources) {
+    Object.keys(dataSources).forEach(key => {
+        const dataSource = dataSources[key];
+        dataSource.connect();
+    });
+}
+
+
+module.exports = app => {
```

Comment:
Minor thing: This function is referenced from the `server/server.js` as `ApplyExecutableMiddleware`. It that's the name we're using there, then we should name this function appropriately here.

Additionally, this function is performing some async operations

* Building the schema (see the `.then`)
* Connecting data sources.

However, the function is made to look like it's synchronous and it does not surface any errors that might happen during either of those operations. So in `server.js` when we call `ApplyExecutableMiddleware(App);` we have no idea if that was successful or not.

The function should either use `async/await` (more preferable) or return a promise.